components/Footer/Footer.tsx

```diff
@@ -0,0 +1,44 @@
+import Image from 'next/image';
+
+import { footerData } from '@/constant';
+
+export default function Footer() {
+	return (
+		<footer>
+			<div className=' w-full border-t bg-white  dark:border-t-black-300 dark:bg-black-200'>
```

Comment:
Be sure to keep in mind the importance of HTML semantics. Perhaps you could have this as a `<section>`